coupon_multi_gift/README.rst

```diff
@@ -0,0 +1,123 @@
+==================
+Coupons multi gift
+==================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsale--promotion-lightgray.png?logo=github
+    :target: https://github.com/OCA/sale-promotion/tree/15.0/coupon_multi_gift
+    :alt: OCA/sale-promotion
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/sale-promotion-15-0/sale-promotion-15-0-coupon_multi_gift
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/296/15.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to define multiple reward products on promotions.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure multiple product rewards:
+
+#. Go to *Sales > Catalog > Coupon Programs* and select or create a new one.
+#. Choose the criteria of appliance you want to use and the minimum quantities, that
+   will be used to calculate the times the rewards can be applied.
+#. On the *Reward Type* field choose *Multi Gift*.
+#. You'll see a now the *Gift list* on which you can add the products you want to give
+   away and the quantities for each of them.
+
+An example:
+
+ ===== ================
+  Qty      Products
+ ===== ================
+    3   Prod A
+    2   Prod B
+ ===== ================
+
+With a minimum quantity of 3, for every 3 units of products that fulfill the domain
+we'd get 3 units of A and 2 of B. So if the valid products quantities are 11, we'd
+get 9 of product A and 6 of product B.
+
+Usage
+=====
+
+This module is base to be used by an extra module that use it in sale orders or the
+point of sale.
+
+Known issues / Roadmap
+======================
+
+* Gifts are applied just once independently from the min quantity set.
```

Comment:
Is this applicable to this module or the sales one?

Reply:
Right. This is for the sales one